libraries/botbuilder-core/src/cloudAdapterBase.ts

```diff
@@ -176,6 +179,82 @@ export abstract class CloudAdapterBase extends BotAdapter {
         return this.processProactive(claimsIdentity, ActivityEx.getContinuationActivity(reference), audience, logic);
     }
 
+    /**
+     * @inheritdoc
+     */
+    async createConversationAsync(
+        botAppId: string,
+        channelId: string,
+        serviceUrl: string,
+        audience: string,
+        conversationParameters: ConversationParameters,
+        logic: (context: TurnContext) => Promise<void>
+    ): Promise<void> {
+        // Create a ClaimsIdentity, to create the connector and for adding to the turn context.
+        const claimsIdentity = this.createClaimsIdentity(botAppId);
+        claimsIdentity.claims.push({ type: AuthenticationConstants.ServiceUrlClaim, value: serviceUrl });
```

Comment:
This implementation is missing the argument validation of the `serviceUrl`, `conversationParameters` and `callback` parameters from [.NET](https://github.com/microsoft/botbuilder-dotnet/blob/001a8a19ba2f4c05ad5149f05689fb6c13495bf4/libraries/Microsoft.Bot.Builder/CloudAdapterBase.cs#L198-L204), is this intentional?

Reply:
Laziness on my part, will fix!